src/nav/style-guide.yml

```diff
@@ -1,119 +1,128 @@
-      - title: Data dictionary style guidelines
-        path: /docs/style-guide/article-templates/data-dictionary-style-guidelines
-      - title: API tutorial template
-        path: /docs/style-guide/article-templates/api-tutorial-template
-      - title: Agent API guide template
-        path: /docs/style-guide/article-templates/agent-api-guide-template
-      - title: apiStyleGuidelines (Example agent API)
-        path: /docs/style-guide/article-templates/apistyleguidelines-example-agent-api
-      - title: GraphQL API tutorial template
-        path: /docs/style-guide/article-templates/graphql-api-tutorial-template
-      - title: Landing page template
-        path: /docs/style-guide/article-templates/landing-page-template
-      - title: Agent release notes template 1.2.3
-        path: /docs/style-guide/article-templates/agent-release-notes-template-123
+    - title: Writing great content at New Relic
+      path: /docs/style-guide/writing-strategies
+      pages: 
+        - title: Introduction to the style guide
+          path: /docs/style-guide/writing-strategies/introduction-style-guide
+        - title: Voice, tone, and style
+          path: /docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic
+    - title: Capitalization
+      path: /docs/style-guide/capitalization
+      pages:
+        - title: Capitalization
+          path: /docs/style-guide/capitalization/capitalization
+    - title: Formatting
+      path: /docs/style-guide/formatting
+      pages:
+        - title: Buttons
+          path: /docs/style-guide/formatting/buttons
+        - title: Callouts
+          path: /docs/style-guide/formatting/callouts      
+        - title: Collapsers
+          path: /docs/style-guide/formatting/collapsers
+        - title: Code references
+          path: /docs/style-guide/formatting/code-examples
+        - title: Code formatting - var and mark
+          path: /docs/style-guide/formatting/code-formatting-guidelines-var-mark
+        - title: Bold or code, not italics
+          path: /docs/style-guide/formatting/bold-or-code-not-italics
+        - title: Heading levels
+          path: /docs/style-guide/formatting/levels-headings
+        - title: Hyperlinks
+          path: /docs/style-guide/formatting/hyperlinks
+        - title: Lists
+          path: /docs/style-guide/formatting/lists
+        - title: Tables
+          path: /docs/style-guide/formatting/tables
+        - title: Formatting terminal commands
+          path: /docs/style-guide/formatting/formatting-terminal-commands
+        - title: Titles, short titles, and paths
+          path: /docs/style-guide/formatting/titles
+        - title: UI paths
+          path: /docs/style-guide/formatting/ui-paths
+    - title: Images, videos, and captions
+      path: /docs/style-guide/images
+      pages:
+        - title: Screenshots and images
+          path: /docs/style-guide/images/screenshots-images 
```

Comment:
I don't understand why the standalone "Embed images" and "Embed videos" docs are not part of the "Images, videos, and captions" category

Reply:
Coming to say the same thing.